api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -7,7 +7,51 @@ These endpoints support the advanced search page and allow for complex filtering
 
 # Group Visualizations
 
-These endpoints return data that is grouped in preset units to support the various data visualizations on USAspending.gov's Advanced Search page.
+These endpoints return data that are grouped in preset units to support the various data visualizations on USAspending.gov's Advanced Search page.
+
+## Spending by Award [/api/v2/search/spending_by_award/]
+
+This endpoint takes award filters and fields, and returns the fields of the filtered awards.
+
+### Spending by Award [POST]
+
++ Request (application/json)
+    + Attributes (object)
+        + filters (optional, FilterObject)
+        + fields: [`Award ID`, `Recipient Name`, `Start Date`, `End Date`, `Award Amount`, `Awarding Agency`, `Awarding Sub Agency`, `Contract Award Type`, `Award Type`, `Funding Agency`, `Funding Sub Agency`] (optional, array[string]),
+            See options at https://github.com/fedspendingtransparency/usaspending-api/blob/stg/usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_award.md#fields
+        + limit: 60 (optional, number)
+            How many results are returned. If no limit is specified, the limit is set to 10.
```

Comment:
We can indicate defaults with API Blueprint format: see https://apiblueprint.org/documentation/mson/specification.html#45-default